From: Your Name <2901318208@qq.com>
Date: Thu, 12 Sep 2025 00:00:01 +0000
Subject: [PATCH 2/2] security: makefile: Add Baseband Guard build dependency

Integrate the Baseband Guard driver into the security subsystem's
build process.

Signed-off-by: Your Name <your.email@example.com>
---
 security/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/security/Makefile b/security/Makefile
index 1234567..abcdefg 100644
--- a/security/Makefile
+++ b/security/Makefile
@@ -10,3 +10,5 @@ obj-$(CONFIG_SECURITY_PATH)		+= path.o
 obj-$(CONFIG_SECURITY_YAMA)		+= yama/
 obj-$(CONFIG_SECURITY_SAFESETID)	+= safesetid/
 obj-$(CONFIG_BPF_LSM)			+= bpf_lsm.o
+
+obj-$(CONFIG_BASEBAND_GUARD)		+= ../drivers/baseband_guard/
